app/controllers/home_controller.rb

```diff
@@ -1,5 +1,11 @@
 class HomeController < ApplicationController
+
+  # TODO: what happens when the user doesn't pertain to any organization?
```

Comment:
I would redirect them to a page where we tell them to join some organization. Do we have something alike @sseerrggii ?